src/main/java/com/liferay/faces/bridge/ext/filter/internal/BridgePortletConfigFactoryLiferayImpl.java

```diff
@@ -32,7 +32,7 @@
 	private BridgePortletConfigFactory wrappedBridgePortletConfigFactory;
 
 	// Instance field must be declared volatile in order for the double-check idiom to work (requires JRE 1.5+)
-	private volatile PortletConfig portletConfig;
+	private transient volatile PortletConfig portletConfig;
```

Comment:
`PortletConfig` is not `Serializable` because it holds references to `ResourceBundle` which is not `Serializable`